### PR TITLE
fix minor issues and add time serie widget

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

- Use ellipsis helper for title layer
image

- Add icon for time series
image

### CODE REVIEW

File: app/assets/stylesheets/editor-3/layers-list.css.scss [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
 .Editor-ListAnalysis-inner {
   @include display-flex();
-  @include flex(1);
+  width: calc(100% + 8px);
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: I'd like to learn why we should use this, please :).

File: lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-category.tpl [hunk 2]
```diff
@@ -6,7 +6,7 @@
     </div>
   </li>
   <li class="WidgetShape-category-item">
-    <p class="WidgetShape-category-title"></p>
+    <p class="WidgetShape-category-titl e"></p>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-category.tpl]: It is possible this was not in purpose ^^^.

Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-category.tpl]: hahahahaha

Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-category.tpl]: Fixed

File: app/assets/stylesheets/editor-3/layers-list.css.scss (hunk 1 above)
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/layers-list.css.scss]: is this ok then @xavijam ? :dancer:

File: app/assets/stylesheets/editor-3/widget-icons.css.scss [hunk 3]
```diff
@@ -89,3 +89,68 @@
   background: $cMainLine;
 }
 
+
+.WidgetShape-TimeSeries-inner {
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/widget-icons.css.scss]: IMHO this class is not correctly named. It should be:

```css
...
.WidgetShape-timeSeries {}
.WidgetShape-timeSeriesInner {}
…
``` 

And checking this name, what is the name of the rest icons?

Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/widget-icons.css.scss]: Ok

.WidgetShape-formula
.WidgetShape-category
.WidgetShape-histogram

File: lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<div class="WidgetShape-timeSeries">
+  <div class="WidgetShape-timeSeries-header">
+    <span class="WidgetShape-timeSeries-play"></span>
+    <h2 class="WidgetShape-timeSeries-title"></h2>
+  </div>
+  <ul class="WidgetShape-timeSeriesInner">
+    <li class="WidgetShape-timeSeries-bar is-played is-01"></li>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl]:
```css
.WidgetShape-timeSeriesBar {}
```

File: lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<div class="WidgetShape-timeSeries">
+  <div class="WidgetShape-timeSeries-header">
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl]: Same as below

File: lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    <span class="WidgetShape-timeSeries-play"></span>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl]: Same as below

File: lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    <h2 class="WidgetShape-timeSeries-title"></h2>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widget-icon-timeSeries.tpl]: Same as below

**Comment by piensaenpixel** [comment]
sorry @xavijam could you take a look :(

**Comment by xavijam** [comment]
f****** master :+1:

**Pull request merged**